app/views/user_mailer/work_package_watcher_removed.html.erb

```diff
@@ -0,0 +1,37 @@
+<%#-- copyright
+OpenProject is a project management system.
+Copyright (C) 2012-2018 the OpenProject Foundation (OPF)
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2017 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See docs/COPYRIGHT.rdoc for more details.
+
+++#%>
+<%= t(:text_work_package_watcher_removed, id: "##{@issue.id}", watcher_setter: @watcher_setter) %>
+<hr />
+<%= render partial: 'issue_details', locals: { issue: @issue } %>
+<p>
+  <%= format_text(t(:text_latest_note, note: last_issue_note(@issue)),
+                  only_path: false,
+                  object: @issue,
+                  project: @issue.project) %>
+</p>
```

Comment:
The only difference between the 'work_package_watcher_added' view and this one is the translation. Extracting the shared code into a partial with the translation key being a parameter would remove the duplication.